snakes/python/summer-league-2022/src/tests.py

```diff
@@ -217,6 +217,23 @@ def test_avoid_single_snake(self):
         self.assertEqual(len(result_moves), 2)
         self.assertEqual(expected, result_moves)
 
- 
+class AvoidWallTest(unittest.TestCase):
+    
+    def test_avoid_wall_top_right(self):
+        # Arrange
+        test_body = [{"x": 10, "y": 10}, {"x": 9, "y": 10}, {"x": 8, "y": 10}, {"x": 7, "y": 10}]
+        # Setup board object with width and height nested objects
+        board = { "width": 10, "height": 10 }
+
+        possible_moves = ["up", "right", "down"]
```

Comment:
I had to leave out left (our neck) here as we are not doing any of the other self-avoidance checks in this test.